### DISCUSSION TITLE
Standard Economy

### DISCUSSION THREAD

**Discussion by MartinAntoniuk** [xanthics/poe_gen_gwennen#20] [General]
Could you please add the possibility to choose the Standard economy of poe.ninja for the regex?

**Comment by xanthics** [comment]
I've taken a look at it.. some work will need to be done for filtering out stuff that can't be chanced as there is A LOT of legacy unique types in Standard.  Current plan is to have SC temp be authoritative and then use it as a filter for the other leagues.

I don't have time right now, but I'll look some more this weekend.

**Reply by MartinAntoniuk** [reply]
Thank you so so much <3 I really appreciate it.

**Comment by xanthics** [comment]
5287a08b3506969299d5ae287a7f20b28b799769

**Reply by MartinAntoniuk** [reply]
You are amazing <3 Thank you so much. Works wonderfully and it will help me, and many others, quite a lot.
